docs/extensions/tutorials/_posts/1970-01-13-UsingLocalization.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/using-localization
+title: Localization
+section: Tutorials
+---
+
+# Localization
+
+i18n stands for internationalization (i, 18 letters, n). Shoutem's localization is based on [i18n-js by fnando](https://github.com/fnando/i18n-js). Each extension has it's own set of translations located in the `app` segment in the `translations` directory. These extension-level translations are fallbacks for when a new extension is made and an existing language file doesn't have it's strings.
+
+Language files are uploaded by the app owner using the `shoutem.i18n` settings page.
+
+<p class="image">
+<img src='{{ site.url }}/img/tutorials/localization/i18n-settings-page.png'/>
+</p>
+
+## Using your Translation
+
+To use your translation, you can download and unzip [this file]({{ site.url }}/static/localization/en.json.zip) and replace all the English strings with anything you like. Either a different language or alternative English strings.
+
+Importing the necessary packages to your extension is very simple:
+
+```JavaScript
+import { i18n } from 'shoutem.i18n';
+```
+
+This of course means that you must have the `shoutem.i18n` extension (Language & Region) extension installed in your app.
+
+The two following entries will explain both how to format your translation (for the app owner and non-technical users) and how to implement translations in the actual code.
+
+#### Pluralized Strings
```

Comment:
This is already an advanced case. Explain above this the simple 'HelloWorld' of translation. So I've imported the i18n, how do I get a string in component translated? Keep it simple first. :) You have it in full example, just show it here first.